.gitignore

```diff
@@ -1,11 +1,24 @@
+# Android Studio / IntelliJ
+/captures
+.externalNativeBuild
+.cxx
 *.iml
-.idea/*
-!.idea/codeStyles
-!.idea/saveactions_settings.xml
+.idea/caches
+.idea/compiler.xml
+.idea/gradle.xml
+.idea/libraries
+.idea/modules.xml
+.idea/workspace.xml
+.idea/navEditor.xml
+.idea/assetWizardSettings.xml
+.idea/deploymentTargetSelector.xml
+
+# Gradle files
 .gradle
-/local.properties
-.DS_Store
 build/
```

Comment:
As we use `build/` here we can safely delete `.gitignore` files in `app`, `core` and `foundation` because these three files only contain `/build`.